shipyard_blueprints/sqlserver/tests/test_auth.py

```diff
@@ -51,3 +51,27 @@ def test_auth_bad_pwd(creds):
         user=creds["user"], pwd="bad_pwd", host=creds["host"], database=creds["db"]
     )
     assert conn_helper(client) == 1
+
```

Comment:
I found that running the cli in a subprocess in pytests is more reliable(I've been bit before). No change needed here but for future tests we should have tests that run the script even though these unit tests are still useful they won't pick up changes to the authtest file